client/components/course/Outline/index.vue

```diff
@@ -72,11 +72,16 @@ export default {
   padding: 80px 90px 0 60px;
   overflow-y: scroll;
   overflow-y: overlay;
+  scrollbar-width: none;
```

Comment:
This is some experimental CSS magic and I'm not really in favor of including it.